src/Confluent.Kafka/Config.cs

```diff
@@ -43,14 +43,35 @@ public class Config : IEnumerable<KeyValuePair<string, string>>
         /// <summary>
         ///     Initialize a new <see cref="Config" /> instance based on
         ///     an existing <see cref="Config" /> instance.
+        ///     <para>
```

Comment:
appreciate that using `<para>` is a step forward, but i haven't used it anywhere else, so i think it would be good to remove these for consistency. does this result in meaningfully better docs anywhere?

Reply:
We use it all over the place. But only really of use if generating MSDN style doc sites.
I'll take them out for now

Reply:
xmldoc consolidated